modules/memory/memory.go

```diff
@@ -28,14 +30,50 @@ import (
 var debug bool = false
 
 type module struct {
+	SandboxProfile sandbox.SandboxProfile
 }
 
 func (m *module) NewRun() modules.Runner {
 	return new(run)
 }
 
+func (m *module) GetSandboxProfile() sandbox.SandboxProfile {
+	return m.SandboxProfile
+}
+
 func init() {
-	modules.Register("memory", new(module))
+	m := new(module)
+	sandbox := sandbox.SandboxProfile{
+		DefaultPolicy: seccomp.ActTrap,
+		Filters: []sandbox.FilterOperation{
+			sandbox.FilterOperation{
+				FilterOn: []string{
+					"openat",
```

Comment:
Same as for file module. in this case i suspect it has to open a bunch of things though, so i wonder if we'll have to filter it later or something like that.